testing/btest/Baseline/scripts.base.protocols.rdp.rdp-invalid-length/analyzer_debug.log

```diff
@@ -3,11 +3,13 @@
 #set_separator	,
 #empty_field	(empty)
 #unset_field	-
-#path	analyzer
+#path	analyzer_debug
 #open XXXX-XX-XX-XX-XX-XX
 #fields	ts	cause	analyzer_kind	analyzer_name	uid	fuid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	failure_reason	failure_data
 #types	time	string	string	string	string	string	addr	port	addr	port	string	string
 XXXXXXXXXX.XXXXXX	violation	protocol	RDP	CHhAvVGS1DHFjwGM9	-	10.0.0.1	45257	10.0.0.2	3389	Binpac exception: binpac exception: &enforce violation : RDP_Negotiation_Response:length	-
+XXXXXXXXXX.XXXXXX	disabled	protocol	RDP	CHhAvVGS1DHFjwGM9	-	10.0.0.1	45257	10.0.0.2	3389	-	-
```

Comment:
Just from skimming some baselines and definitely can be follow-up to avoid extra diffs: What would you think about having `aid: count &log &optional` added to `analyzer_debug.log` ? You mentioned that would also be useful for file analyzers, even if we don't have it right now.

This seems to be an example of enabling RDP 3 times on the same connection, A separate aid column would further show that it's 3 different instances of the analyzer.

Reply:
I would probably put that into a follow-up PR, but... I like it.